src/applications/vaos/covid-19-vaccine/redux/selectors.js

```diff
@@ -13,11 +13,11 @@ export function selectCovid19Vaccine(state) {
 }
 
 export function selectCovid19VaccineNewBooking(state) {
-  return selectCovid19Vaccine(state).newBooking;
+  return selectCovid19Vaccine(state)?.newBooking;
```

Comment:
`newBooking` should always exist, if it doesn't, I think something is wrong (or a test might not be setting redux state data correctly).